source/skyconnect/index.html

```diff
@@ -177,12 +177,23 @@ <h1 class="sub-title">FAQs</h1>
-  Nothing changes for current users of the multiprotocol firmware who are happy with their experience. The experimental multiprotocol firmware will remain available, but we will not recommend it to new users. Instead, we will focus on making sure the dedicated Zigbee and Thread firmwares for Home Assistant SkyConnect deliver the best experience to users.{% enddetails %}
+  As part of our efforts to bring Matter to Home Assistant, we're now focused on ensuring that the Thread experience will catch up and become a first-class citizen, making it easy to connect to your Thread-based Matter devices in Home Assistant without a third-party Thread border router. The Thread firmware is already fully functional under the hood, but we still have some work to do to make the experience of using Thread-based devices in Home Assistant feel good. Once the experience has improved, we will recommend using this Thread firmware to power your Thread network as an alternative to using third-party Thread border routers from Apple or Google.
+  {% enddetails %}
+
+  {% details "What is the current state of multiprotocol support?" %} 
+  <br />
+  <b>Multiprotocol will remain an experimental feature and is not recommended for use. </b>
+  <br /><br />
+  When we launched Home Assistant SkyConnect, we announced our intent to release firmware supporting multiprotocol, which allows the device's Silicon Labs chip to connect to both Zigbee and Thread networks with one radio. <br /><br />
+  This experimental firmware has been available since December 2022. Through extensive testing, we have found that although it works in some circumstances, it has technical limitations that lead to a worse user experience. We now do not recommend using this firmware, and it will be experimental for the foreseeable future. Instead, we will focus on making sure the dedicated Zigbee and Thread firmwares for Home Assistant SkyConnect deliver the best experience to users.<br /><br />
+  
+  If you currently have the multiprotocol firmware installed but don't actively use it to connect to Thread devices, we recommend that you <a href="https://skyconnect.home-assistant.io/procedures/disable-multiprotocol/">disable multiprotocol</a>. Nothing changes for current users of the multiprotocol firmware who are happy with their experience. The experimental multiprotocol firmware will remain available, but we will not recommend it to new users. <br /><br />
+  Nothing changes for current users of the multiprotocol firmware who are happy with their experience. The experimental multiprotocol firmware will remain available, but we will not recommend it to new users.{% enddetails %}
```

Comment:
The "Nothing changes for current users [...]" part seems to be duplicated.

Screenshot of preview: 
![image](https://github.com/home-assistant/home-assistant.io/assets/6409465/33c96653-5c84-4756-969b-a7a1796a4063)

Reply:
Oooh, good catch @TheJulianJES!

Reply:
https://github.com/home-assistant/home-assistant.io/pull/32872